pkg/virt-operator/resource/generate/components/crds.go

```diff
@@ -635,6 +635,12 @@ func NewKubeVirtCR(namespace string, pullPolicy corev1.PullPolicy, featureGates
 		}
 	}
 
+	if infraReplicas != 2 {
```

Comment:
Maybe it would be better to have a `const` somewhere. Hard-coding "2" will make it harder to change in the future.